examples/src/main/scala/org/apache/spark/examples/ml/GeneralizedLinearRegressionExample.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// scalastyle:off println
+package org.apache.spark.examples.ml
+
+// $example on$
+import org.apache.spark.ml.regression.GeneralizedLinearRegression
+// $example off$
+import org.apache.spark.sql.SparkSession
+
+object GeneralizedLinearRegressionExample {
```

Comment:
I like the comment being there - we should try to have that consistently in
all the examples
On Tue, 10 May 2016 at 17:07, Seth Hendrickson <notifications@github.com>
wrote:

> In
> examples/src/main/scala/org/apache/spark/examples/ml/GeneralizedLinearRegressionExample.scala
> <https://github.com/apache/spark/pull/12754#discussion_r62689150>:
>
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +// scalastyle:off println
> > +package org.apache.spark.examples.ml
> > +
> > +// $example on$
> > +import org.apache.spark.ml.regression.GeneralizedLinearRegression
> > +// $example off$
> > +import org.apache.spark.sql.SparkSession
> > +
> > +object GeneralizedLinearRegressionExample {
>
> In other examples, we place a comment at the top of the file explaining
> how to run the example. For instance:
>
> /** * An example demonstrating a bisecting k-means clustering. * Run with * {{{ * bin/run-example ml.BisectingKMeansExample * }}} */
>
> We seem to be very inconsistent about this. It would be nice to choose a
> convention and stick to it. Thoughts? @MLnick <https://github.com/MLnick>
> @zhengruifeng <https://github.com/zhengruifeng>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12754/files/faa9892f7b6fcc245456d3b83ca4f8d03a220b64#r62689150>
>